core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.html

```diff
@@ -48,8 +48,8 @@ <h3 class="subtopic-card-heading">Review Material</h3>
             <span class="form-text error-msg" *ngIf="errorMsg">
               <em>{{ errorMsg }}</em>
             </span>
-            <span class="form-text error-message" *ngIf="editableTitle?.length === 80">
-              <em>The maximum length of subtopic title is 80 characters.</em>
+            <span class="form-text error-message" *ngIf="editableTitle?.length === 64">
```

Comment:
The `MAX_CHARS_IN_SUBTOPIC_TITLE` constant is set to 64(So, I haven't really changed it), as can be seen in the [assets/constants.ts](https://github.com/oppia/oppia/blob/develop/assets/constants.ts#L6200) , and the error message is displayed only once the user has reached the maximum number of characters allowed in the input field. Therefore, it should match the max characters permitted, which is 64.